src/v2/Apps/Consign/Components/SellWithArtsy.tsx

```diff
@@ -2,8 +2,26 @@ import React from "react"
 import { Box, Button, Flex, Image, Link, Text, color } from "@artsy/palette"
 import { SectionContainer } from "./SectionContainer"
 import { Media } from "v2/Utils/Responsive"
+import { useTracking } from "react-tracking"
+import { ContextModule, OwnerType, clickedAppDownload } from "@artsy/cohesion"
+
+const DOWNLOAD_URL =
```

Comment:
(non-blocking) Not part of the scope of this work but feels like we should just make this an app-wide constant since the URL should always be unique.